graphql-server/Makefile

```diff
@@ -1,3 +1,14 @@
+SIGNATURE_SECRET := $(shell openssl rand -hex 64)
+.PHONY: secret
+secret:
+	cp -p .env .env.$$(date +%Y%m%d%H%M%S).bak || true
+	cp .env.example .env
+	sed -i '' -e "s/__SIGNATURE_SECRET__/$(SIGNATURE_SECRET)/g" ".env"
```

Comment:
FYI, `sed -i ''` is not working on GNU sed. So this will be better


```
sed -i.new -e "s/__SIGNATURE_SECRET__/$(SIGNATURE_SECRET)/g" ".env.example"
mv .env.example.new .env
```

Reply:
this seems to copy .env.example to .env and modify the original .env.example for me tho 🤔

Reply:
since `-i` creates a backup `.env.example.new` would be the original `.env.example` in this case?

Reply:
Yes, you are right, I am wrong.

Reply:
i think will replace with `sed -e "s/__SIGNATURE_SECRET__/$(SIGNATURE_SECRET)/g" ".env.example" > .env`